sdk/textanalytics/Azure.AI.TextAnalytics/README.md

```diff
@@ -86,10 +86,12 @@ A **text input**, sometimes called a **document**, is a single unit of input to
 An operation result, such as `AnalyzeSentimentResult`, is the result of a Text Analytics operation, containing a prediction or predictions about a single text input.  An operation's result type also may optionally include information about the input document and how it was processed.
 
 ### Operation Result Collection
- An operation result collection, such as `AnalyzeSentimentResultCollection`, is a collection of operation results, where each corresponds to one of the text inputs provided in the input batch.  A text input and its result will have the same index in the input and result collections.  An operation result collection may optionally include information about the input batch and how it was processed.
+An operation result collection, such as `AnalyzeSentimentResultCollection`, is a collection of operation results, where each corresponds to one of the text inputs provided in the input batch.  A text input and its result will have the same index in the input and result collections.  An operation result collection may optionally include information about the input batch and how it was processed.
 
- ### Operation Overloads
- For each supported operation, `TextAnalyticsClient` provides method overloads to take a single text input, a batch of text inputs as strings, or a batch of either `TextDocumentInput` or `DetectLanguageInput` objects.  The overload taking the `TextDocumentInput` or `DetectLanguageInput` batch allows callers to give each document a unique ID, or indicate that the documents in the batch are written in different languages.
+ ### Operation on multiple text inputs
+For each supported operation, `TextAnalyticsClient` provides a method that accepts a batch of text inputs as strings, or a batch of either `TextDocumentInput` or `DetectLanguageInput` objects. This methods allow callers to give each document a unique ID, or indicate that the documents in the batch are written in different languages.
+
+**Note:** Multiple text inputs methods are the expected methods to use when working on a production environment.
```

Comment:
Could we say batching methods instead of "multiple text inputs"? Might also want to add why they should be used, e.g. better performance, etc.